docs/program/run.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Run SDK Code"
+linkTitle: "Run Code"
+weight: 50
+type: "docs"
+description: "Execute the logic you've written to control your robot or fleet."
+images: ["/services/img/icons/sdk.svg"]
+tags: ["client", "sdk", "application"]
+---
+
+After saving your boilerplate code sample and adding control logic with Viam's API methods, run your program to control your Viam-connected robot.
+Do this by running a command to execute the program in the terminal of a machine with the appropriate programming language and Viam SDK installed.
```

Comment:
I feel like this was covered previously?  Is this page meant to repeat that, or is it supposed to be about running your code in different ways (connecting over the web vs locally, etc)?

Reply:
It's providing context for what you will have done before getting to this page.